File: src/atv30/Calculadora.java
/**
 * Calculadora
 */
public class Calculadora {

    private int numUm;
    private int numDois;

    public int getNumUm() {
        return numUm;
    }
    public void setNumUm(int numUm) {
        this.numUm = numUm;
    }
    public int getNumDois() {
        return numDois;
    }
    public void setNumDois(int numDois) {
        this.numDois = numDois;
    }

    public int soma(){
        int soma = numUm + numDois;
        return soma;
    }

    public int subtracao(){
        int subtracao = numUm - numDois;
        return subtracao;
    }

    public int multiplicacao(){
        int multiplicacao = numUm * numDois;
        return multiplicacao;
    }

    public double divisao(){
        double divisao = numUm / numDois;
        return divisao;
    }



    
}

File: src/atv70/Calculadora.js
class Calculadora{
    numeroUm;
    numeroDois;


    somar(numeroUm, numeroDois){
        return numeroUm + numeroDois;
    }

    subtracao(numeroUm, numeroDois){
        return numeroUm - numeroDois;
    }

    multiplicacao(numeroUm, numeroDois){
        return numeroUm * numeroDois;
    }

    divisao(numeroUm, numeroDois){
        return numeroUm / numeroDois;
    }
    
}

console.log("testando calculadora teste");

const calc = new Calculadora();
const soma = calc.somar(1,1)
const sub = calc.subtracao(2,2)
const mult = calc.multiplicacao(2,2)
const div = calc.divisao(4,4)

console.log("Soma: ",soma)
console.log("Subtração: ",sub)
console.log("Multiplicação: ",mult)
console.log("Divisão: ",div)